fs/rails-base-graphql-api
# Update gems

### Summary

- Update rails and related action* gems to 6.0.4.8
  - Update sprockets to 4.0.3
  - Update rack to 2.2.3.1
  - Update loofah to 2.18.0
  - Update nokogiri to 1.13.6
  - Update concurrent-ruby to 1.1.10
- Update puma to 5.6.4
### How it works

N/A

### Test plan

List of steps to manually test introduced functionality:

* Setup project
* Run `bin/quality`
* Make sure there's no warnings 
* Make sure app may run correctly

### Review notes

While reviewing pull-request (especially when it's your pull-request),
please make sure that:

- you understand what problem is solved by PR and how is it solved
- new tests are in place, no redundant tests
- DB schema changes reflect new migrations
- newly introduced DB fields have indexes and constraints
- there are no missed files (migrations, view templates)
- required ENV variables added and described in `.env.example` and added to Heroku
- associated Heroku review app works correctly with introduced changes

### Deploy notes

N/A

### References
[CVE-2022-22577](https://groups.google.com/g/ruby-security-ann/c/NuFRKaN5swI)
[CVE-2022-27777](https://groups.google.com/g/ruby-security-ann/c/9wJPEDv-iRw)
[CVE-2022-21831](https://groups.google.com/g/rubyonrails-security/c/n-p-W1yxatI)
[Integer Overflow or Wraparound in libxml2 affects Nokogiri](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-cgx6-hpwq-fhv5)
[CVE-2022-24839](https://github.com/sparklemotion/nekohtml/security/advisories/GHSA-9849-p7jc-9rmv)
[CVE-2022-23437](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-xxx9-3xcr-gjj3)
[CVE-2018-25032](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-v6gp-9mmm-c6p5)
[CVE-2022-29181](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-xh29-r2w5-wx8m)
[CVE-2022-24836](https://github.com/sparklemotion/nokogiri/security/advisories/GHSA-crjr-9rc5-ghw8)
[CVE-2022-30123](https://groups.google.com/g/ruby-security-ann/c/LWB10kWzag8)
[CVE-2022-30122](https://groups.google.com/g/ruby-security-ann/c/L2Axto442qk)
